sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/internal/state/MetricStorageUtilsTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.state;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.entry;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.mock;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.metrics.internal.aggregator.Aggregator;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class MetricStorageUtilsTest {
+
+  private static final Attributes a = Attributes.of(AttributeKey.stringKey("a"), "a");
+  private static final Attributes b = Attributes.of(AttributeKey.stringKey("b"), "b");
+  private static final Attributes c = Attributes.of(AttributeKey.stringKey("c"), "c");
+  private static final Attributes d = Attributes.of(AttributeKey.stringKey("d"), "d");
+  private static final Attributes e = Attributes.of(AttributeKey.stringKey("e"), "e");
+  private Map<Attributes, String> result;
+  private Map<Attributes, String> toMerge;
+
+  @BeforeEach
+  private void setup() {
+    result = new HashMap<>();
+    result.put(a, "A");
+    result.put(b, "B");
+    result.put(d, null);
```

Comment:
We shouldn't allow `null` accumulations.

Reply:
There's nothing to prevent a user from passing in a map with null values, though, and because there definitely is code that checks for null values, I think it makes sense to have coverage in that area.